baseline.yaml

```diff
@@ -129,18 +129,20 @@ criteria:
     maturity_level: 1
     category: Build & Release
     criteria: |
-      The project's build and release pipelines
-      MUST NOT execute arbitrary code that is
-      input from outside of the build script.
+      A build process that either leads to release or can read secret data
+      MUST NOT execute untrusted code.
```

Comment:
We'd also need to cover any build pipelines that produce artifacts that would be used on a system that might contain sensitive information, such as an end user or developer machine. 🤔 Effectively any pipeline that produces any artifact that could be used by any machine that isn't ephemeral.

Reply:
Would this work?: s/secret/sensitive/